spec/components/search_result/document_component_spec.rb

```diff
@@ -3,13 +3,26 @@
 require "rails_helper"
 
 RSpec.describe SearchResult::DocumentComponent, type: :component do
-  pending "add some examples to (or delete) #{__FILE__}"
-
-  # it "renders something useful" do
-  #   expect(
-  #     render_inline(described_class.new(attr: "value")) { "Hello, components!" }.css("p").to_html
-  #   ).to include(
-  #     "Hello, components!"
-  #   )
-  # end
+  let(:component) { described_class.new(document: presenter) }
+  let(:view_context) { double('ViewContext', action_name: 'index', document_index_view_type: 'list', blacklight_config: ArticlesController.blacklight_config) }
```

Comment:
Can you use `vc_test_controller.view_context` instead?  You may need to wrap `render_inline` with `with_controller_class(CatalogController) do ... end`

Reply:
done